### ISSUE TITLE
High Contrast Themes Inverted Object Status Styles

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1036]
**Describe the bug**
In the Fiori 3 high contrast themes Inverted Object Status is not styled according to description.
Also, object Status with Generic Indication Colors do not have text color that corresponds to the theme.

> What version of the Fundamental Library Styles are you using?
HCB v0.9.1-rc.0
HCW v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/object-status.html and https://deploy-preview-1025--fundamental-styles.netlify.app/components/object-status.html
2. The Inverted Object Status text and background colors are inconsistent with description 

**Expected behavior**
Inverted Object Status should have semantically colored backgrounds and black (for HCB) or white (HCW) text color.

Object Status with Generic Indication Colors should have appropriate text-color i.e. black for HCB and white for HCW.

**Screenshots**
HCW
![image](https://user-images.githubusercontent.com/43764832/81861188-b403eb00-951c-11ea-9e12-5a52f9a62e0f.png)
![image](https://user-images.githubusercontent.com/43764832/81861488-2a085200-951d-11ea-91d8-6674e5ba2742.png)

HCB
![image](https://user-images.githubusercontent.com/43764832/81861202-ba926280-951c-11ea-904e-1b72d7889f1f.png)
![image](https://user-images.githubusercontent.com/43764832/81861507-3391ba00-951d-11ea-92d0-e4dff9426c16.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - HCW and HCB now have correct colors.
<img width="1108" alt="Screen Shot 2020-10-08 at 5 59 58 PM" src="https://user-images.githubusercontent.com/21978807/95529586-24aa1900-0990-11eb-9f5e-3a3a78c12154.png">
<img width="865" alt="Screen Shot 2020-10-07 at 4 56 08 PM" src="https://user-images.githubusercontent.com/21978807/95529591-283da000-0990-11eb-94d5-840ff8a054ab.png">
<img width="773" alt="Screen Shot 2020-10-08 at 6 00 55 PM" src="https://user-images.githubusercontent.com/21978807/95529619-3c819d00-0990-11eb-810a-82b3040930dc.png">
<img width="777" alt="Screen Shot 2020-10-07 at 4 59 43 PM" src="https://user-images.githubusercontent.com/21978807/95529621-3f7c8d80-0990-11eb-8e3c-9dbbe6c4d9f9.png">
